src/components/layouts/MobileLayoutUsageExample.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * This file is an example usage of the MobileLayout component.
+ * You can use this file as a reference when you need to use the MobileLayout component.
+ *
+ * This component should be used at the root level of a page component,
+ * located in the "src/components/page/" directory.
+ */
+
+'use client';
+
+import { DeleteIcon, MenuCircleIcon, PenIcon } from '@/assets/images/icons';
+import { MobileLayout } from '@/components/layouts/MobileLayout';
+import { HeaderBackButton } from '@/components/parts/Header';
+import {
+  Button,
+  DropdownMenu,
+  DropdownMenuButton,
+  DropdownMenuButtonIcon,
+  DropdownMenuButtonText,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+} from '@/components/ui';
+
+import { DropdownMenuPortal } from '@radix-ui/react-dropdown-menu';
+
+export const MobileLayoutUsageExample = () => {
+  return (
+    <MobileLayout
+      heading="Groups"
+      headerLeft={<HeaderBackButton href={{ pathname: '/groups' }} />}
+      headerRight={<HeaderDropdownMenu />}
+    >
+      MobileLayoutUsageExample
+    </MobileLayout>
+  );
+};
+
+const HeaderDropdownMenu = () => {
+  const handleRenameClick = () => {
+    alert('Rename clicked!');
+  };
+
+  const handleDeleteClick = () => {
+    alert('Delete clicked!');
+  };
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        {/* Replacing this button with 'HeaderMenuCircleButton' causes this dropdown menu to stop working */}
+        <Button className="h-full px-4 flex items-center">
+          <Icon icon={MenuCircleIcon} size={6} color="black" />
+        </Button>
```

Comment:
I found out this problem happened because `HeaderMenuCircleButton` component is not forward ref.
Since the component is not forwardref, but just a functional component,  `asChild` in `DropdownMenuTrigger` is not working properly.
With current `HeaderMenuCircleButton`, I got this error ⬇️  when I wrapped it with `DropdownMenuTrigger`.
<img width="377" alt="Screenshot 2024-02-22 at 19 07 41" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99339182/63b2acb0-c204-4a03-a25e-a813f9833f86">

After changing `HeaderMenuCircleButton` into forwardref like this ⬇️ , it started working properly in my environment. 
```tsx
export const HeaderMenuCircleButton = forwardRef<
  HTMLButtonElement,
  ButtonHTMLAttributes<HTMLButtonElement>
>(({ ...props }, ref) => (
  <Button className="h-full px-4 flex items-center" ref={ref} {...props}>
    <Icon icon={MenuCircleIcon} size={6} color="black" />
  </Button>
));

HeaderMenuCircleButton.displayName = 'HeaderMenuCircleButton';
```